src/main/java/com/doyak/reflector/auth/handler/JwtAuthenticationEntryPoint.java

```diff
@@ -0,0 +1,35 @@
+package com.doyak.reflector.auth.handler;
+
+import java.io.IOException;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.AuthenticationEntryPoint;
+import org.springframework.stereotype.Component;
+
+import com.doyak.reflector.payload.ApiResponse;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Component
+public class JwtAuthenticationEntryPoint implements AuthenticationEntryPoint {
+
+    @Override
+    public void commence(HttpServletRequest request, HttpServletResponse response,
+                         AuthenticationException authException) throws IOException {
+
+        log.error("Not Authenticated Request", authException);
+        ApiResponse<Object> apiResponse =
+                ApiResponse.onFailure("COMMON401", HttpStatus.UNAUTHORIZED.name(), "인증이 필요합니다.");
```

Comment:
```suggestion
        ErrorStatus errorStatus = ErrorStatus.UNAUTHORIZED;
        ApiResponse<Object> apiResponse = ApiResponse.onFailure(errorStatus.getCode(), errorStatus.getHttpStatus().name(), errorStatus.getMessage());
```
ErrorStatus enum 사용해서 응답 구성하는 건 어떨까요?

Reply:
💡 좋습니다!